corehq/apps/geospatial/reports.py

```diff
@@ -8,12 +8,11 @@
 from corehq.apps.reports.standard.cases.data_sources import CaseDisplayES
 from corehq.apps.reports.standard.cases.case_list_explorer import CaseListExplorer
 from .const import GEO_POINT_CASE_PROPERTY
-from .models import GeoPolygon
+from .models import GeoPolygon, GeoConfig
 
 
-
-def _get_geo_location(case):
-    geo_point = case['case_json'].get(GEO_POINT_CASE_PROPERTY)
+def _get_geo_location(case, geo_data_prop=GEO_POINT_CASE_PROPERTY):
```

Comment:
We could add a single method such as `get_geo_case_property` in a utils.py that returns the configured property or default property, so the callers can directly get that property without worrying to use `GEO_POINT_CASE_PROPERTY` by default.